src/components/Swiper/index.tsx

```diff
@@ -20,27 +20,40 @@ export interface SwiperProps extends SwiperOptions {
 
 const generateId = createUniqIDGenerator('swiper-');
 
-export const Swiper: FC<SwiperProps> = props => {
+export const Swiper: FC<SwiperProps> = ({
+  getSwiperInstance,
+  className,
+  children,
+  navigationChildren,
+  paginationChildren,
+  ...swiperOptions
+}) => {
   const swiperRef: React.MutableRefObject<OriginalSwiper | null> = useRef<OriginalSwiper>(null);
   const containerId = useMemo(() => generateId(), []);
 
   useEffect(() => {
```

Comment:
useEffect 2개끼리 중복되는 코드가 있어서 한개만 쓰고 이렇게 쓰는게 더 깔끔할듯
```suggestion
  useEffect(() => {
    swiperRef.current = new OriginalSwiper(`#${containerId}`, swiperOptions);
    if (getSwiperInstance) {
      getSwiperInstance(swiperRef.current);
    }
    return () => {
      if (swiperRef.current) {
        swiperRef.current.destroy(true, true);
      }
    };
  }, [containerId, swiperRef, swiperOptions, getSwiperInstance]);
```

Reply:
수정완료띠